requirements.txt

```diff
@@ -0,0 +1,4 @@
+numpy
```

Comment:
Not now, but we need to discuss version locking

Reply:
definitely!